packages/state-transition/package.json

```diff
@@ -53,7 +53,7 @@
     "lint": "biome check src/ test/",
     "lint:fix": "yarn run lint --write",
     "test": "yarn test:unit",
-    "test:unit": "vitest run --project unit",
+    "test:unit": "vitest run --project unit-minimal --project unit-mainnet",
```

Comment:
on current unstable branch it's not possible to run state-transition tests alone because we moved them to `unit-mainnet` folder but the `--project unit` just checks `unit` folder, this is really error prone if you have to think about this per package so I just added both `unit-minimal` and `unit-mainnet` in each command so if someone in the future adds tests to either of these they will be executed